### PR TITLE
Add docs for discord source

### PR DESCRIPTION
**Pull request by Licenser**

### CODE REVIEW

File: docs/artefacts/onramps.md [hunk 1]
```diff
@@ -318,6 +318,8 @@ onramp:
 
 ### rest
 
+**This source can be linked**
```
Comment (mfelsche) [inline docs/artefacts/onramps.md:321]: can we stick with one naming here that is `onramp`? Just for now. IIRC we are going to do the great renaming once we have connectors.

File: docs/artefacts/onramps.md [hunk 2]
```diff
@@ -435,6 +437,8 @@ config:
 
 ### ws
 
+**This source can be linked**
```
Comment (mfelsche) [inline docs/artefacts/onramps.md:440]: same here

File: docs/artefacts/onramps.md [hunk 3]
```diff
@@ -467,3 +471,98 @@ onramp:
       port: 12201
       host: "127.0.0.1"
 ```
+
+### discord
+
+**This source can be linked**
```
Comment (mfelsche) [inline docs/artefacts/onramps.md:477]: and here :)

File: docs/artefacts/onramps.md [hunk 4]
```diff
@@ -467,3 +471,98 @@ onramp:
...
+
+The `discord` connector allows consuming events from the [Discord API](https://discord.com/developers/docs/intro). It uses the event structure as provided by [serenity](https://docs.rs/serenity/0.10.2/serenity/) wrapped in event-named records.
```
Comment (mfelsche) [inline docs/artefacts/onramps.md:479]: connector -> onramp

**Review by mfelsche — COMMENTED** [review]
Some small naming stuff, otherwise awesome!

File: docs/artefacts/onramps.md (hunk 1 above)
Comment (Licenser) [reply docs/artefacts/onramps.md]: good point all updated!

**Review by mfelsche — APPROVED** [review]

**Pull request merged**